tests/integration/it_utils.py

```diff
@@ -124,6 +143,22 @@ async def sign_and_reliable_submission_async(
     return response
 
 
+def send_timed_reliable_submission(
+    transaction: Transaction, use_json_client: bool = True
+) -> Response:
+    client = _choose_client(use_json_client)
+    SyncTestTimer(1, client.request, (LEDGER_ACCEPT_REQUEST,)).start()
+    return send_reliable_submission_sync(transaction, client)
+
+
+async def send_timed_reliable_submission_async(
+    transaction: Transaction, use_json_client: bool = True
+) -> Response:
+    client = _choose_client_async(use_json_client)
+    AsyncTestTimer(1, client)
```

Comment:
You never call start on this one, is that intentional?

Reply:
Yes, the AsyncTestTimer starts on initialization